README.md

```diff
@@ -1,70 +1,58 @@
-# Getting Started with Create React App
+# Country-Facts
+Microverse Module 03 Project: This educational project is to create a React Country-Facts Application that allows the user to display various metric information about a given selected Country from an API.
+ 
+## Built With the following
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- CSS
+- React.js
+- Javascript
+- JSX
+- Babel
+- Redux
+- Linters
 
-## Available Scripts
+## Live Demo
+[Live Demo Link](Coming soon)
```

Comment:
- [ ] you should according to the requirements deploy the application using Heroku, Netlify or GitHub Pages.